rebasehelper/tests/test_utils.py

```diff
@@ -629,3 +630,39 @@ def test_get_macros(self):
         assert macros[0]['name'] == 'test_macro'
         assert macros[0]['value'] == 'test_macro value'
         assert macros[0]['level'] == -1
+
+
+class TestLookasideCacheHelper(BaseTest):
+
+    SETUPS = [
+        {
+            'tool': 'fedpkg',
+            'package': 'vim-go',
+            'filename': 'v1.6.tar.gz',
+            'hashtype': 'md5',
+            'hash': '847d3e3577982a9515ad0aec6d5111b2',
+            'url': 'http://pkgs.fedoraproject.org/repo/pkgs',
+        },
+        pytest.mark.xfail(run=False, reason='this works only inside Red Hat internal network')
+        ({
+            'tool': 'rhpkg',
+            'package': 'libtiff',
+            'filename': 'tiff-3.9.4.tar.gz',
+            'hashtype': 'md5',
+            'hash': '2006c1bdd12644dbf02956955175afd6',
+            'url': 'http://pkgs.devel.redhat.com/repo/pkgs',
```

Comment:
I don't think the internal infrastructure URLs should be in public code. Please remove it.